package.json

```diff
@@ -7,7 +7,7 @@
     "lint": "standard",
     "lint:fix": "standard --fix",
     "redis": "docker run -p 6379:6379 --rm redis",
-    "test": "standard && tap test/*.test.js && npm run typescript",
+    "test": "standard && tap && npm run typescript",
```

Comment:
By default tap search for tests in the test folder that have the extension `.test.js` (or other criteria), I've removed the explicit files location as it wouldn't work for the location of the new test - `/test/github-issues/<file>`, I've could add another glob (`test/**/*.test.js`) but instead, I removed it